js/5etools-backgrounds.js

```diff
@@ -424,34 +459,96 @@ function d20plusBackgrounds () {
 		let ideal = null;
 		let bond = null;
 		let flaw = null;
+		const matchCharacteristics = [
+			"Suggested Characteristics", // Most backgrounds
```

Comment:
this I'm fine with, but it could maybe be a slightly-more-accepting regex `/\bcharacteristics\b/i.test(ent.name)` to catch any other minor variations. I doubt backgrounds should have other headers with "Characteristics" in them

Reply:
Did a little research, seems like includes() is a bit faster, so I used that instead of the regex.